src/components/app/types.ts

```diff
@@ -11,6 +11,7 @@ export interface AddNewAppProps extends RouteComponentProps<{}> {
 export interface OptionType {
     label: string
     value: string
+    format?: string
```

Comment:
extend `format` type instead

Reply:
We should not add the format in generic type. we should extend this for our specific use case